tests/test_scene.py

```diff
@@ -53,6 +53,45 @@ def test_cosmos_basic():
     except ImportError:
         print 'The assert_raises tests require nose'
 
+    # Try making galaxies
+    gal_real = cat2.makeGalaxy(index=0,gal_type='real',chromatic=False)
+    if not isinstance(gal_real, galsim.RealGalaxy):
+        raise TypeError("COSMOS Catalog makeGalaxy routine does not return an instance of"+\
+                        "'galsim.RealGalaxy'")
```

Comment:
Ah, I didn't know that.

If there are no more comments, I'll merge it by tonight.
On Apr 17, 2016 4:55 PM, "Mike Jarvis" <notifications@github.com> wrote:

> In tests/test_scene.py
> <https://github.com/GalSim-developers/GalSim/pull/740#discussion_r59992520>
> :
>
> > @@ -53,6 +53,45 @@ def test_cosmos_basic():
> >      except ImportError:
> >          print 'The assert_raises tests require nose'
> >
> > +    # Try making galaxies
> > +    gal_real = cat2.makeGalaxy(index=0,gal_type='real',chromatic=False)
> > +    if not isinstance(gal_real, galsim.RealGalaxy):
> > +        raise TypeError("COSMOS Catalog makeGalaxy routine does not return an instance of"+\
> > +                        "'galsim.RealGalaxy'")
>
> You don't need the \ here. Or the + for that matter. Python automatically
> concatenates two consecutive strings.
>
> However, you should put a space at the end of the first string. Otherwise
> you'll get "... of'galsim.RealGalaxy'".
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/GalSim-developers/GalSim/pull/740/files/1d814b617fb5b84dc1adbd77ad575717b84b51ca#r59992520>
>